### PR TITLE
Move virtual env of all examples to main folder

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

Move the repeated virtual env setup section in all example's readme to the main readme, standardize to two requirements files within each example folder

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
Thanks for the effort, please don't change cifar10 example in this PR.
I will submit a PR that fix this and move cifar10

File: examples/virtualenv/set_env.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export projectname='nvflare_cifar10'
+export projectname='venv_nvflare_example'
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/virtualenv/set_env.sh:3]: sure

**Comment by ZiyueXu77** [comment]
> 

sure, reversed changes to cifar

File: examples/advanced/brats18/README.md [hunk 2]
```diff
@@ -96,9 +72,9 @@ cd ./result_stat
 bash testing_models_3d.sh
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/brats18/README.md:72]: thanks for the suggestion! we can apply a general fix in another PR

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM for this one.
We can do some other re-factors later.

**Pull request merged**